Cargo.toml

```diff
@@ -10,13 +14,70 @@ build = "build.rs"
 edition = "2018"
 
 [dependencies]
-cli = { package = "node-cli", path = "node/cli" }
+log = "0.4"
```

Comment:
Hmm, don't we maybe want to clean this up and just keep the members in the root `Cargo.toml`.

The following should help for `cargo run`:
https://github.com/rust-lang/cargo/issues/7032

(I'm not sure if this already included in stable or not.

Reply:
It's not in stable.

Reply:
Can we just wait and postpone until this is in stable?

Reply:
You mean around 2022

Reply:
This is already stabilized, so I would assume that it is released with the next rust release? https://github.com/rust-lang/cargo/pull/7056